dev/MenuBar/MenuBar_TestUI/MenuBarPage.xaml

```diff
@@ -55,6 +55,8 @@
                     <MenuFlyoutItem Text="Word Wrap"/>
                     <MenuFlyoutItem Text="Font..."/>
                 </muxc:MenuBarItem>
+
+                <muxc:MenuBarItem Title="No children"/>
```

Comment:
Thanks @chingucoding. Could we add a test ? I think you can click and look at all the open popups to validate.

Reply:
Added new interaction test now.